docs/cloud/serverless.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Serverless 
+layout: default
+parent: Cloud
+has_children: false
+nav_order: 14 
+---
+
+# Serverless
+*Currently in preview*
+
+## Why Serverless?
+Serverless is a new architecture to FeatureBase that separates compute power and storage. This allows for independent scaling of compute, storage, or both.  
+
+## Creating a Serverless Database
```

Comment:
Heading should be Before you begin

```
## Before you begin

* [Review how to create a Cloud database](/docs/cloud/cloud-databases/cloud-db-create) because of similarities with this process.
```

When you're creating hyperlink anchor text it needs to be meaningful so it's picked up by search and by users. So "See here", "here" and related words aren't any use anymore.